### PR TITLE
Priydes/validate migration

### PR DESCRIPTION
**Pull request by priydes**
Add support for validate migration api

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@priydes__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by huangpf** [comment]
LGTM.

File: lib/services/storageManagement/lib/storageManagementClient.js [hunk 1]
```diff
@@ -2004,5 +2004,134 @@ var StorageAccountOperations = ( /** @lends StorageAccountOperations */ function
...
+   * Classic to ARM migration.
+   * 
+   * @param {string} storageAccountName  Name of storage account to be migrated.
+   * 
+   * @param {function} callback
+   * 
+   * @returns {Stream} The response stream.
+   */
+  StorageAccountOperations.prototype.validateMigration = function (storageAccountName, callback) {
+        if (callback === null || callback === undefined) {
```
Comment (amarzavery) [inline lib/services/storageManagement/lib/storageManagementClient.js]: was this hand written or autogenerated.
Everything else in the file above has two spaces indentation whereas the new addition has 4 spaces. Can you please indent it to 2 spaces.

**Comment by amarzavery** [comment]
Please make sure to publish packages for all the 3 services.

File: lib/services/storageManagement/lib/storageManagementClient.js (hunk 1 above)
Comment (priydes) [inline lib/services/storageManagement/lib/storageManagementClient.js]: Fixed. Auto-generated the code now. I was not aware of code auto-generator. Thanks!

**Comment by azurecla** [comment]
Hi __@priydes__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by amarzavery** [comment]
Since the CI  is hung and this is required urgently, we shall merge this right now.

**Pull request merged**